src/application/tester.go

```diff
@@ -37,24 +44,40 @@ func TestApp(appContext types.AppContext, logger *util.Logger, mode composebuild
 	}
 	if numFailed == 0 {
 		logger.Log("All tests passed")
-		return true, nil
+		return true, false, nil
 	}
 	logger.Logf("%d tests failed", numFailed)
-	return false, nil
+	return false, false, nil
 }
 
-// TestService runs the tests for the service and return true if the tests passed
+// TestService runs the tests for the service and return true if the tests passed, if they were interrupted,
 // and an error if any
-func TestService(serviceContext types.ServiceContext, logger *util.Logger, mode composebuilder.BuildMode) (bool, error) {
+func TestService(serviceContext types.ServiceContext, logger *util.Logger, mode composebuilder.BuildMode) (bool, bool, error) {
```

Comment:
Can we create a `TestResult` struct which has passed and interrupted fields instead of returning two bools? Same for `TestApp`